src/eddymotion/math/utils.py

```diff
@@ -43,3 +43,31 @@ def is_positive_definite(matrix):
     except np.linalg.LinAlgError:
         # Matrix is not positive definite
         return False
+
+
+def compute_angle(v1, v2, closest_polarity=False):
+    """Compute the angle between two vectors.
+
+    Parameters
+    ----------
+    v1 : :obj:`~numpy.ndarray`
+        First vector.
+    v2 : :obj:`~numpy.ndarray`
+        Second vector.
+    closest_polarity : :obj:`bool`
+        ``True`` to compute the smallest angle between the two vectors.
+
+    Returns
+    -------
+    :obj:`float`
+        Angle between vectors.
+    """
```

Comment:
Let's add a quick unit test:

```suggestion
    
    Examples
    --------
    >>> compute_angle(
    ...     np.array((1.0, 0.0, 0.0)),
    ...     np.array((-1.0, 0.0, 0.0)),
    ... )  # doctest: +ELLIPSIS
    3.1415...
    >>> compute_angle(
    ...     np.array((1.0, 0.0, 0.0)),
    ...     np.array((-1.0, 0.0, 0.0)),
    ...     closest_polarity=True,
    ... )
    0.0
    
    """
```

Reply:
Added in d69bafd.